### DISCUSSION TITLE
Is it possible to Ignore "missing an opposite relation field" error?

### DISCUSSION THREAD

**Discussion by AssisrMatheus** [prisma/prisma#9309] [Q&A] [answered]
I'm making a key-value translation table:
```prisma
model LocalizedText {
  key      String @id @db.VarChar(255)
  value    String @db.Text
  language String @db.VarChar(10)

  @@unique([key, language], name: "unique_key_language")
}
```

Where other models can access:
```prisma
model Collection {
  id             Int           @id @default(autoincrement())

  nameKey        String        @db.VarChar(255)
  name           LocalizedText @relation("Collection_name_LocalizedText", fields: [nameKey], references: [key])

  descriptionKey String        @db.VarChar(255)
  description    LocalizedText @relation("Collection_description_LocalizedText", fields: [descriptionKey], references: [key])
}
```

But I keep getting the following error:
```sh
Error validating field `name` in model `Collection`: The relation field `name` on Model `Collection` is missing an opposite relation field on the model `LocalizedText`. Either run `prisma format` or add it manually.
```

**I know what it's for and why the error occurs**, but I'm in a situation where it doesn't matter for me to have the opposite relation field in my types, and it would actually pollute my `LocalizedText ` model with tons of fields. Is there a way to ignore this warning and not have the opposite fields on `LocalizedText `?

---

Edit: I realized that with this logic each model can only have one localization, which defeats the purpose of having multiple languages, and I have updated my schema to account for that.

But my question still remains, is there a way to ignore this error, and prevent prisma plugin from generating the opposite field automatically?

**Comment by ryands17** [comment, marked as answer]
@AssisrMatheus 👋 

Unfortunately that's not possible at the moment. You always need to have opposite relations as that makes querying on both sides possible.

You can create a feature request for this [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this.

**Reply by AssisrMatheus** [reply]
Alright. Thank you!

**Comment by dderiso** [comment]
Are single-sided relations supported now?

**Reply by ludralph** [reply]
Hi @dderiso 

No they are not supported at the moment.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
